.github/workflows/deploy-prd-enhanced-worker.yml

```diff
@@ -1,6 +1,6 @@
 on:
   push:
-    paths: ["src/**", "cdk-infra/lib/constructs/worker/**"]
+    paths: ["src/**", "cdk-infra/lib/constructs/worker/**", "Dockerfile.enhanced", "cdk-infra/utils"]
```

Comment:
Just a heads up that pushes for paths aren't evaluated for pushes to tags, based on the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore). 🤔 

> Path filters are not evaluated for pushes of tags.

Reply:
Good catch, @rayangler! I will just remove the paths for the `prd` actions which shouldn't be an issue

Reply:
Yeah, it's interesting behavior, but makes sense. We can definitely apply this path filter to a pre-prod environment in the future